docs/source/troubleshooting/troubleshooting-driver-plugins.adoc

```diff
@@ -134,6 +134,28 @@ $ sudo virsh undefine minishift
 
 In case all of this fails, you might want to xref:../getting-started/uninstalling.adoc#[uninstall Minishift] and do a fresh install of Minishift.
 
+[[Network-not-found-no-network-with-matching-name-default]]
+=== Network not found: no network with matching name 'default'
+
+At present, the KVM driver doesn't attempt to create the default network - it assumes it's already present. If you create a VM with another tool like virt-manager first it will create it (or you can create it from the command line.) Try the following to create the network
+
```

Comment:
To run Minishift successfully on KVM, it needs the `default virtual network` provided by libvirt.The default virtual network usually gets created by automatically for a standard installation. However in case it is not present in the host you will see the discussed error. You can use the following commands to create the default network.

Reply:
"gets created by automatically for a standard installation." I do not think this is needed, as this makes people starting to guess WHY it didn't set it up. most likely as they didn't use virtmanager, etc. It creates ambiguity, so I would leave it out.